frontend/src/components/Profile/MyFriends.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-card :loading="loading" class="ma-5" max-width="400">
+    <v-card-title class="white--text orange darken-4">
+      Friends
+
+      <v-spacer></v-spacer>
+
+      <v-btn color="white" class="text--primary" fab small>
+        <v-icon>mdi-plus</v-icon>
+      </v-btn>
+    </v-card-title>
+
+    <v-virtual-scroll :items="users" item-height="50" height="300" width="500">
+      <template v-slot:default="{ item }">
+        <v-list-item>
+          <v-list-item-content>
+            <v-list-item-title>{{ item.username }}</v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </template>
+    </v-virtual-scroll>
+  </v-card>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  data() {
+    return {
+      loading: true,
+      users: [],
+    };
+  },
+  async created() {
+    let response = await axios.get("/friendships/", {
+      params: {
+        sourceId: this.$store.getters.id,
+        status: "accepted",
```

Comment:
If I'm not mistaken, it will be necessary to get all friendships where `sourceId = id || targetId = id`. But I imagine @N-4-Nina can confirm or deny.

Also PR #113 might be relevant here (there is a route for getting friendships or friend requests directly through the user instead)

Reply:
Absolutely, /users/friendships is what we want here i think, but I'm in the process of rewriting the Profile view myself, so it does not matter much right now.

Reply:
Okay, sounds good!